sqllogictest-bin/src/lib.rs

```diff
@@ -61,11 +62,13 @@ struct Opt {
     junit: Option<String>,
 
     /// The database server host.
+    /// If multiple addresses are specified, one will be chosen randomly per session.
     #[clap(short, long, default_value = "localhost")]
-    host: String,
+    host: Vec<String>,
     /// The database server port.
+    /// If multiple addresses are specified, one will be chosen randomly per session.
     #[clap(short, long, default_value = "5432")]
-    port: u16,
+    port: Vec<u16>,
```

Comment:
It looks weird. We should merge them into one argument:
```suggestion
    /// If multiple addresses are specified, one will be chosen randomly per session.
    #[clap(short, long, default_value = "localhost:5432")]
    host: Vec<String>,
```

Reply:
I've thought of this, but this change will break the command-line interfaces. Is this acceptable? 😢

Reply:
Oops. How about introducing a new argument?
```rust
    /// Multiple addresses to database servers.
    /// If specified, one will be chosen randomly per session. --host and --port will be ignored.
    #[clap(long, default_value = "localhost:5432")]
    hosts: Vec<SocketAddr>,
```

Reply:
Actually the `tokio-postgres` interface is also designed to accept multiple hosts and ports separately. 😄

https://docs.rs/tokio-postgres/0.7.7/src/tokio_postgres/connect.rs.html#22-24

Reply:
Is that mean this feature has already been supported by `tokio-postgres` so we can directly forward `host` and `port` to it?

Reply:
`tokio-postgres` will try to connect to each address in order if an error occurs, which differs from our demands. 😢

Reply:
😢. Anyway, since `tokio-postgres` is designed this way, it looks acceptable to me.